tfhe-zk-pok/src/proofs/pke.rs

```diff
@@ -186,14 +189,26 @@ impl<G: Curve> PublicParams<G> {
     deserialize = "G: Curve, G::G1: serde::Deserialize<'de>, G::G2: serde::Deserialize<'de>",
     serialize = "G: Curve, G::G1: serde::Serialize, G::G2: serde::Serialize"
 ))]
-#[versionize(PKEv1ProofVersions)]
+#[versionize(ProofVersions)]
```

Comment:
Why rename?

Reply:
Before everything was in the `backward_compatibility/mod.rs`, so I I have to name the types differently to avoid conflicts. 

But for the old versions this lead to names such as `PKEv1ProofV0` and `PKEv2ProofV0`, which are a bit confusing. So I split the `backward_compatibility/mod.rs` into `backward_compatibility/pke.rs` and `backward_compatibility/pke_v2.rs` to mirror what is done in the main module.